src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,84 @@
+#SFMC Credentials
+admin.clientid = SALESFORCE_MARKETING_CLIENT_ID
+admin.clientsecret =SALESFORCE_MARKETING_CLIENT_SECRET
+admin.base.uri=SALESFORCE_MARKETING_BASE_URI
+admin.soap.endpoint=SALESFORCE_MARKETING_SOAP_API_ENDPOINT
+admin.rest.endpoint = SALESFORCE_MARKETING_REST_API_ENDPOINT
+
+#SFMC User -Invalid Credentials
+invalid.clientid=a231
+invalid.clientsecret=564avf
+invalid.authenticationbase.uri=http//authenticationbase.com/
+invalid.soapapi.endpoint=http//invalidsoapi/
+invalid.restapibase.uri=http//invalidrespapi
+
+filter.value=folderId = 27860
+StoresId=xLko431
+
+#macro properties
+singleobjectmode.objectname=Mailing List
+multiobjectmode.objectlist=Mailing List
+singleobjectmode.dataextensionkey=Stores
+multiobjectmode.dataextensionkeylist=key1,key2
+multiobjectmode.tablename=Table1
+
+
+#Sink properties
+INSERT=insert
+UPDATE=update
+UPSERT=upsert
+TRUE=true
+FALSE=false
+sink.dataextensionkey=key1234
+sink.maxbatchsize=1000
+sink.columnmapping=inputfiled,dataextension
+
+#bq properties
+projectId=cdf-athena
+datasetprojectId=cdf-athena
+dataset=test_automation_sfmc
+bqtarget.table=target-table
+bqsource.table=source-table
+multisource.table=mailinglist
+
+#File path
+expectedOutputFile = testdata/expected_outputs/MailingListOutputFile
+expectedOutputFile1 = testdata/expected_outputs/MailingListOutputFileWithFilter
+
+
+#schema json
+sfmcSourceSchema.bounceevent=[{"key":"id","value":"string"},{"key":"smtpCode","value":"string"},\
```

Comment:
please make sure we test all supported data types in source and sink. e.g: double, decimal

Reply:
Added the data type double, for decimal the team is looking for an SObject which supports decimal data type. (https://github.com/data-integrations/salesforce-marketing/pull/34/commits/5fdfef427e89af7a0d44b3ec552e491572211beb#diff-fbf79cedac53a4c6cd803eab1cb931f681b8f6038521c562463760671182d0caR57)